packages/frontend/editor-ui/src/components/NodeSettings.vue

```diff
@@ -186,7 +170,7 @@ const executeButtonTooltip = computed(() => {
 });
 
 const nodeVersionTag = computed(() => {
-	if (!props.nodeType || props.nodeType.hidden) {
+	if (!nodeType.value || nodeType.value) {
```

Comment:
Are you missing `hidden` here?
```suggestion
	if (!nodeType.value || nodeType.value.hidden) {
```